### PR TITLE
Fix #268 - Add documentation about QR Code Generator to application d…

### PR DESCRIPTION
**Pull request by EstherPhebe**
…esigner docs

closes odk-x/tool-suite-X#268

#### What is included in this PR?
1. Added documentation about QR Code Generator to [Application Designer docs](https://docs.odk-x.org/app-designer-using/)
2. Removed images of the old application designer interface and replaced them with images of the current interface.

### CODE REVIEW

File: odkx-src/app-designer-overview.rst [hunk 1]
```diff
@@ -70,17 +71,27 @@ File Browser
...
+.. image:: /img/app-designer-overview/appscan-form-designer.*
+  :alt: The ODK-X Scan Form Designer tab in the ODK-X Application Designer
 
 See :doc:`scan-form-designer-intro` documentation for more information about this tool.
 
+.. _app-designer-overview-qr:
+
+ODK-X QR Code Generator
+--------------------------
+The :guilabel:`QR Code Generator` tab contains the QR Code Generator, a tool that generates a QR Code to log into the synced Server . The Server URL, username and password details are entered into their specified input fields, then a QR Code is generated.
```
Comment (elmps2018) [inline odkx-src/app-designer-overview.rst:92]: some minor proof reading issues here -- server can be lower case and there's an extra space before the period that can be cut.

**Review by elmps2018 — COMMENTED** [review]
Thanks for working on this! I spotted just one spot for minor editing otherwise looks great!

File: odkx-src/app-designer-overview.rst (hunk 1 above)
Comment (EstherPhebe) [reply odkx-src/app-designer-overview.rst:92]: Thanks for reviewing. I'll fix that now.

**Comment by elmps2018** [comment]
@EstherPhebe awesome thanks for quick fix!

**Pull request merged**